app/models/apple/podcast_container.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Apple
-  class PodcastContainer < ActiveRecord::Base
+  class PodcastContainer < ApplicationRecord
```

Comment:
In rails 7, these should all extend our `ApplicationRecord`, not `ActiveRecord::Base`.  Guessing these just got missed when brought over from CMS.